docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  registry:
+    image: registry:2
+    network_mode: host
+  localstack:
+    image: localstack/localstack
```

Comment:
looks like localstack's ECR support is not supported in the OSS version, isn't it? https://github.com/localstack/localstack#overview

Reply:
oh, sorry I missed it isn't provided in the community version. I'll remove it and write unit and e2e tests with a local registry.